pkg/logging/logging.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) The Tellor Authors.
+// Licensed under the MIT License.
+
+package logging
+
+import (
+	"os"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/pkg/errors"
+	"github.com/tellor-io/telliot/pkg/config"
+)
+
+// NewLogger create a new logger
```

Comment:
Please follow Golang commentary guidelines.
https://golang.org/doc/effective_go.html#commentary

Here and few other places
```suggestion
// NewLogger create a new logger.
```